xml/art_migration.xml

```diff
@@ -395,8 +396,20 @@ done</screen>
     <itemizedlist>
      <listitem>
       <para>
-       For non-DVR clouds, evacuate any existing &o_netw; L3 routers on the
-       seed node:
+       In Distributed Virtual Router (DVR) enabled clouds, responsibility for
+       network routing is shared across the compute nodes, and downtime of the
+       seed node as a controller will have no adverse effect on workload network
+       traffic. To see if DVR is enabled in your cloud, run the following command
+       and check the entries in the "Distributed" column.
+      </para>
+<screen><prompt>(old-deployer)> </prompt>source ~/service.osrc
+<prompt>(old-deployer)> </prompt>neutron router-list</screen>
+      <para>
+       If you observe non-distributed routers, then the next step will ensure
+       that any responsibility for those routers' function is migrated away from
+       the seed node to other nodes in the controller cluster, so that routing
+       remains available even while the seed node is taken offline. (This step
+       is not necessary if all your routers are distributed.)
```

Comment:
Similarly, could we say "if all your routers are distributed or no Centralized Virtual Routers are running on the seed node" ?

Reply:
See above, I prefer preserving the current content.